public_html/update_builds_html.pl

```diff
@@ -76,6 +76,7 @@ sub add_platform {
 add_platform("ds", "ds", "Nintendo DS", "0", "tar.xz");
 add_platform("gc", "gamecube", "Nintendo Gamecube", "0", "tar.xz");
 add_platform("wii", "wii", "Nintendo Wii", "0", "tar.xz");
+add_platform("switch", "switch", "Nintendo Switch", "0", "zip");
```

Comment:
Having a switch platform icon in `scummvm-web` would be nice for the page to look good.

Reply:
fixed in 83633e16ecd7e0094b120ffb0fbc324593126ea7

Reply:
And in https://github.com/scummvm/scummvm-web/commit/74c6ead665e94cc907e739edbc8c6bae86679648

I didn't realize the icon appears in two places.